app/upload/views.py

```diff
@@ -36,8 +35,8 @@ def upload_document(service_id):
         return jsonify(error='Value for is_csv must be a boolean'), 400
 
     mimetype = get_mime_type(file_data)
-    if mimetype not in chain(*current_app.config['ALLOWED_FILE_TYPES'].values()):
-        allowed_file_types = ', '.join(sorted(f"'.{x}'" for x in current_app.config['ALLOWED_FILE_TYPES'].keys()))
+    if mimetype not in current_app.config['ALLOWED_FILE_TYPES'].keys():
```

Comment:
shouldn't need this `.keys()` call as `x in dict` just does that check anyway

Reply:
Good spot, I've emended that in the commit 👍.